Firestore/CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Unreleased
 
-# v1.11.3
-- [changed] Internal changes.
+# v1.12.1
+- [changed] Internal improvements for future C++ and Unity support.
+
+# v1.12.0
+- [changed] Internal improvements.
```

Comment:
I remember this change being difficult to document, but if you're able to succinctly specify why this version was a minor version increase instead of a patch version increase, it would be good to add here.

Reply:
Took a crack at this, but ultimately it comes down to: this was breaking for (alpha) C++ but not for CocoaPods/Carthage/etc users.